.env.template

```
@@ -0,0 +1,12 @@
+MONGODB_URI=
+
+REDIS_HOST=
+REDIS_PORT=
+REDIS_USERNAME=
+REDIS_PASSWORD=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The REDIS_PASSWORD key should go before the REDIS_PORT key

(UnorderedKey)